OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -429,35 +434,37 @@ void populate_contactModelPrimitives(SimTK::Array_<DecorativeGeometry>& stdPrimi
 
 }
 
-void populate_wrapModelPrimitives(SimTK::Array_<DecorativeGeometry>& stdPrimitives) {
+void populate_wrapModelPrimitives(SimTK::Array_<DecorativeGeometry>& stdPrimitives, bool includeFrames) {
     stdPrimitives.clear();
     // Frame for Ground & Bodies
-    stdPrimitives.push_back(
-        DecorativeFrame(1.0).setBodyId(0).setColor(SimTK::White)
-        .setIndexOnBody(0).setScale(0.2).setOpacity(1)
-        .setRepresentation(SimTK::DecorativeGeometry::DrawSurface));
-    stdPrimitives.push_back(
-        DecorativeFrame(1.0).setBodyId(1).setColor(SimTK::White)
-        .setIndexOnBody(0).setScale(0.2).setOpacity(1)
-        .setRepresentation(SimTK::DecorativeGeometry::DrawSurface));
-    stdPrimitives.push_back(
-        DecorativeFrame(1.0).setBodyId(2).setColor(SimTK::White)
-        .setIndexOnBody(0).setScale(0.2).setOpacity(1)
-        .setRepresentation(SimTK::DecorativeGeometry::DrawSurface));
-    stdPrimitives.push_back(
-        DecorativeFrame(1.0).setBodyId(4).setColor(SimTK::White)
-        .setIndexOnBody(0).setScale(0.2).setOpacity(1)
-        .setRepresentation(SimTK::DecorativeGeometry::DrawSurface));
-    stdPrimitives.push_back(
-        DecorativeFrame(1.0).setBodyId(3).setColor(SimTK::White)
-        .setIndexOnBody(0).setScale(0.2).setOpacity(1)
-        .setRepresentation(SimTK::DecorativeGeometry::DrawSurface));
-    // Frames for the Joint
-    stdPrimitives.push_back(
-        DecorativeFrame(1.0).setBodyId(1).setColor(SimTK::White)
-        .setIndexOnBody(0).setScale(0.2).setOpacity(1)
-        .setRepresentation(SimTK::DecorativeGeometry::DrawSurface)
-        .setTransform(Vec3({-.01, -.03, .03})));
+    if (includeFrames) {
+        stdPrimitives.push_back(
```

Comment:
Perhaps cleaner to use a `for` loop here? …though I see that all you did was indent the existing code.